src/mediawiki/pages/npc/__snapshots__/npc.test.ts.snap

```diff
@@ -52,6 +66,20 @@ exports[`npcPageBuilder should build npc page with transcript 1`] = `
 "
 `;
 
+exports[`npcPageBuilder should build npc page without chathead when no chatheadModels 1`] = `
+"{{New Content}}
+{{Infobox Monster
+|name = name
+|image = [[File:name.png|120px]]
+|members = Yes
+|combat = 1
+|examine = 
+|id = 1
+}}
+
```

Comment:
This space should not be here. The `MediaWikiBreak` right below the if statement for checking for `chatheadModels` in `npc.ts` should belong under the `new MediaWikiFile` for the chathead.